ParseCSV/ParseCSV/Helper.cs

```diff
@@ -46,96 +46,11 @@ public static void CreateCsvFile(string path, string[] collunms_name, double[] v
         // реализует консольный ввод требуемых значений (путь до файла-источника, год, месяц, путь до файла с результатами)
         public static InputData GetInputData()
         {
-            bool isPathInputFileCorrect = false;
-            bool isPathOutputFileCorrect = false;
-            bool isYearCorrect = false;
-            string pathOutputFile;
-            int year;
-            int month;
             var output = new InputData();
```

Comment:
I think variable name should say about input, not output

Reply:
Just used to call returned data like that. I will fix it.